### DISCUSSION TITLE
Latency Types source

### DISCUSSION THREAD

**Discussion by arbuzz202** [nkaz001/hftbacktest#92] [Q&A] [answered]
Hi, saw this awesome diagram about latencies in trading in your documentation and was wondering about it's source (if it wasn't created by you)?
<img width="708" alt="Screenshot 2024-07-02 at 3 25 53 PM" src="https://github.com/nkaz001/hftbacktest/assets/11259231/68a28071-ed88-4f64-adcf-ffb204e2a793">

**Comment by nkaz001** [comment, marked as answer]
I made that diagram.
